cypress/integration/checkout/checkout.multipiece.items.spec.js

```diff
@@ -0,0 +1,136 @@
+import TestTypes from '../../support/dictionary/testTypes';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import getRandomPostfix from '../../support/utils/stringTools';
+import Helper from '../../support/fragments/finance/financeHelper';
+import CheckOutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import NewServicePoint from '../../support/fragments/service_point/newServicePoint';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints';
+import ConfirmMultipieceCheckOutModal from '../../support/fragments/checkout/confirmMultipieceCheckOutModal';
+import UsersEditPage from '../../support/fragments/users/usersEditPage';
+import Checkout from '../../support/fragments/checkout/checkout';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+
+describe('Check Out', () => {
+  let user = {};
+  let userBarcode;
+  const instanceTitle = `autotest_instance_title_${getRandomPostfix()}`;
+  const testItems = [];
+  const defautlDescription = `autotest_description_${getRandomPostfix()}`;
+  let servicePoint;
+  let materialTypeName;
+  let testInstanceIds;
+
+  beforeEach(() => {
+    cy.getAdminToken().then(() => {
+      cy.getLoanTypes({ limit: 1 });
+      cy.getMaterialTypes({ limit: 1 })
+        .then(({ id, name }) => {
+          materialTypeName = { id, name };
+        });
+      cy.getLocations({ limit: 2 });
+      cy.getHoldingTypes({ limit: 2 });
+      cy.getInstanceTypes({ limit: 1 });
+    }).then(() => {
+      const getTestItem = (numberOfPieces, hasDiscription, hasMissingPieces) => {
+        const defaultItem = {
+          barcode: Helper.getRandomBarcode(),
+          status:  { name: 'Available' },
+          permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+          materialType: { id: materialTypeName.id },
+        };
+        if (numberOfPieces) {
+          defaultItem.numberOfPieces = numberOfPieces;
+        }
+        if (hasDiscription) {
+          defaultItem.descriptionOfPieces = defautlDescription;
+        }
+        if (hasMissingPieces) {
+          defaultItem.numberOfMissingPieces = 2;
+          defaultItem.missingPieces = defautlDescription;
+        }
+        return defaultItem;
+      };
+      testItems.push(getTestItem(1, true, false));
+      testItems.push(getTestItem(3, true, false));
+      testItems.push(getTestItem(2, true, true));
+      testItems.push(getTestItem(1, false, true));
```

Comment:
from TC in TestRail - "and some value for number of missing pieces and/or description of missing pieces" - not implemented yet

Reply:
It is implemented in 56 and 57 line